pages/common/uniq.md

```diff
@@ -0,0 +1,12 @@
+# uniq
+
+> Output the unique lines from the given input or file
+> Compare the adjacent files in input (file or standard input) and outputs unique lines
+
+- Finding unique words in a given file
+
+`sort words.txt | uniq`
+
+- Display the count for the unique items with -c
```

Comment:
Good point, but I think count is more precise. Total can mean sum, which may be confusing.